### PR TITLE
Issue 28 ADDS: version cmd line arg

### PR DESCRIPTION
**Pull request by bwagner**
Version is shown in help, but it's useful to have version as a separate command line option.

### CODE REVIEW

File: pdd [hunk 1]
```diff
@@ -422,6 +422,8 @@ def get_parser():
            help='quiet mode for background timer/stopwatch')
     addarg('keywords', nargs='*', help='diff/add/subtract from today or now')
 
+    addarg('-v', '--version', action='version', version=f"%(prog)s version {_VERSION_}")
```
Comment (jarun) [inline pdd:425]: Please add to the README and completion scripts as well.

**Review by jarun — CHANGES_REQUESTED** [review]

File: pdd (hunk 1 above)
Comment (bwagner) [reply pdd:425]: Sorry that it wasn't complete! I pushed the changes. I hope they are included in the original PR now.

File: auto-completion/bash/pdd.bash [hunk 2]
```diff
@@ -12,6 +12,7 @@ _pdd () {
         --add
         -h --help
         -q --quiet
+        -v --version
```
Comment (jarun) [inline auto-completion/bash/pdd.bash:15]: Please add at the end.

File: auto-completion/fish/pdd.fish [hunk 3]
```diff
@@ -9,6 +9,7 @@ complete -c pdd -s d  -l date      -r --description 'calculate date difference'
 complete -c pdd       -l day       -r --description 'show day of the week on a date'
 complete -c pdd -s h  -l help         --description 'show help'
 complete -c pdd -s q  -l quiet        --description 'quiet mode for background timer/stopwatch'
+complete -c pdd -s v  -l version      --description 'show version'
```
Comment (jarun) [inline auto-completion/fish/pdd.fish:12]: Please add at the end.

**Review by jarun — CHANGES_REQUESTED** [review]

**Comment by jarun** [comment]
Also, what is the output?

**Comment by bwagner** [comment]
> Also, what is the output?

pdd version 1.7

File: auto-completion/fish/pdd.fish (hunk 3 above)
Comment (bwagner) [reply auto-completion/fish/pdd.fish]: done.

**Comment by jarun** [comment]
Just print `1.7`.

**Review by jarun — APPROVED** [review]

**Comment by jarun** [comment]
Thank you!

**Comment by bwagner** [comment]
My pleasure and thanks for your patience!

**Comment by jarun** [comment]
No problem! Do let me know if you have any new features for `pdd` in mind.

**Pull request merged**